src/components/Club.js

```diff
@@ -0,0 +1,10 @@
+import NightlifeIcon from "@mui/icons-material/Nightlife";
+
+export function Club(){
```

Comment:
instead of having 1 component per suit, it might have been better to have a single `CardIcon` component that receives a suit as props and renders the correct icon:

```jsx
<CardIcon suit={suit} />
```

Reply:
within the `CardIcon` component you could also create some kind of "icon map" to map the suit name to the icon:

```jsx
import HeartBrokenIcon from "@mui/icons-material/HeartBroken";
import NightlifeIcon from "@mui/icons-material/Nightlife";

const ICON_MAP = { 
  hearts: <HeartBrokenIcon fontSize="large" />
  clubs: <NightlifeIcon fontSize="large" />,
  // other suits here
};

export function CardIcon(suit) {
  return ICON_MAP[suit];
}
```

Reply:
ah I see. I tried this and it works. thanks!